examples/hosts

```diff
@@ -41,4 +41,3 @@
 # leading 0s:
 
 ## db-[99:101]-node.example.com
-
```

Comment:
@Akasurde @evitalis could you please revert this part of the patch? `git checkout 43d6fdc~1 -- examples/hosts && git commit examples/hosts` should fix it, but this needs to be done locally, I cannot fix it via GitHub UI.